cypress/e2e/addBook.cy.js

```diff
@@ -0,0 +1,47 @@
+import addBook from '../fixtures/addBook.json'
+
+describe('Browsed Collection', () => {
+  beforeEach(() => {
+    cy.visit('localhost:3000/add')
+  })
+
+    it('should be able to search a book by name or author', () => {
+      cy.get('[data-cy="add-book-title"]').should('be.visible')
```

Comment:
nit. instead of using .should('be.visible'), consider replacing it with a .contains('Your Page Title Here')